### PR TITLE
Sync changes from release-package fork V1.1.1

### PR DESCRIPTION
**Pull request by danmcpherson**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Tendocs Documents/apiDefinition.swagger.json [hunk 1]
```diff
@@ -61,16 +58,103 @@
...
+          },
+          "404": {
+            "description": "Service has not been found, please contact support@tendocs.com"
+          },
+          "500": {
+            "description": "Something has gone wrong, please contact support@tendocs.com"
+          }
+        },
+        "summary": "Create a new document from a template and JSON",
+        "description": "Create a new Office document by merging an office template with JSON Obeject."
```
Comment (vmanoharas) [inline certified-connectors/Tendocs Documents/apiDefinition.swagger.json]: Please correct the typo "Obeject"

File: certified-connectors/Tendocs Documents/apiDefinition.swagger.json [hunk 2]
```diff
@@ -336,22 +474,235 @@
...
+          "type": "string",
+          "example": "Summary of the document provided.",
+          "x-ms-summary": "Summary",
+          "description": "Summary of the document."
+        },
+        "targetWords": {
+          "type": "number",
+          "example": 300,
+          "x-ms-summary": "Target",
+          "description": "The targetted length in words."
```
Comment (vmanoharas) [inline certified-connectors/Tendocs Documents/apiDefinition.swagger.json]: Please correct the typo "targetted"

File: certified-connectors/Tendocs Documents/apiDefinition.swagger.json [hunk 3]
```diff
@@ -557,6 +908,315 @@
...
+          "description": "Full URL to a logo that is displayed in the top left of the document.",
+          "x-ms-visibility": "advanced",
+          "x-ms-summary": "Document Logo",
+          "example": {
+            "logoUrl": "https://app.tendocs.com/images/doc.png"
+          }
+        },
+        "email": {
+          "type": "string",
+          "description": "Email address of the document recipeint.",
```
Comment (vmanoharas) [inline certified-connectors/Tendocs Documents/apiDefinition.swagger.json]: Please correct the typo

File: certified-connectors/Tendocs Documents/apiDefinition.swagger.json [hunk 4]
```diff
@@ -557,6 +908,315 @@
...
+          "description": "The date after which the document is no longer valid and cannot be accessed.",
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Expiry Date",
+          "example": {
+            "expiry": "2024-06-01T09:00:00Z"
+          }
+        },
+        "isSignatureRequired": {
+          "type": "boolean",
+          "description": "Determines if the recipents name and an empty text box where they need to enter their name is included on the document page.",
```
Comment (vmanoharas) [inline certified-connectors/Tendocs Documents/apiDefinition.swagger.json]: Please correct the recipents

File: certified-connectors/Tendocs Documents/apiDefinition.swagger.json [hunk 5]
```diff
@@ -557,6 +908,315 @@
...
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Signature Required",
+          "example": {
+            "isSignatureRequired": true
+          }
+        },
+        "organisationTitle": {
+          "type": "string",
+          "description": "The title of the company, department or organisation that is repsonsible for sending the document.",
```
Comment (vmanoharas) [inline certified-connectors/Tendocs Documents/apiDefinition.swagger.json]: Please correct the type "repsonsible"

File: certified-connectors/Tendocs Documents/apiDefinition.swagger.json [hunk 6]
```diff
@@ -557,6 +908,315 @@
...
+          "x-ms-visibility": "important",
+          "x-ms-summary": "Organsiation Title",
```
Comment (vmanoharas) [inline certified-connectors/Tendocs Documents/apiDefinition.swagger.json]: Please correct the typo "Organsiation"

File: certified-connectors/Tendocs Documents/apiDefinition.swagger.json [hunk 7]
```diff
@@ -557,6 +908,315 @@
...
+          "example": {
+            "organisation": "Human Resources"
+          }
+        },
+        "organisationWebsite": {
+          "type": "string",
+          "description": "Link to a webiste. This could be internal, or external to the organisation and provide more details on the document or process.",
```
Comment (vmanoharas) [inline certified-connectors/Tendocs Documents/apiDefinition.swagger.json]: Please correct the type "webiste"

File: certified-connectors/Tendocs Documents/apiDefinition.swagger.json [hunk 8]
```diff
@@ -557,6 +908,315 @@
...
+          "x-ms-summary": "Organisation Owner",
+          "example": {
+            "organisationContactName": "Pauline Jones"
+          }
+        },
+        "organisationContactTitle": {
+          "type": "string",
+          "description": "Job title of the owner who is responsible for the document or process.",
+          "x-ms-visibility": "advanced",
+          "x-ms-summary": "Oragnisation Owner Title",
```
Comment (vmanoharas) [inline certified-connectors/Tendocs Documents/apiDefinition.swagger.json]: Please correct the typo "Oragnisation"

**Review by vmanoharas — COMMENTED** [review]
Hello @danmcpherson, 

Kindly address review comments. Additionally, please see the swagger validation errors below and confirm that the existing flows would break or not?

##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/V1ConversionConvertPost/parameter/request/schema/configuration/documentResponseFormat Source: NewSwagger
##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/V1AiSummaryPost/parameter/request/schema/configuration/targetWords Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryPost/responses/200/schema/Type Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryPost/responses/200/schema/Content Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryPost/responses/200/schema/Length Source: NewSwagger
##[error]The path template has been changed for the operation. ValuePath: operation/V1AiSummaryResult Source: NewSwagger
##[error]The required parameter has been added. ValuePath: operation/V1AiSummaryResult/parameter/jobId Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryResult/responses/200/schema/Type Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryResult/responses/200/schema/Content Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryResult/responses/200/schema/TargetWords Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryResult/responses/200/schema/Length Source: NewSwagger
##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/V1AiComparePost/parameter/request/schema/sourceDocument Source: NewSwagger
##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/V1AiComparePost/parameter/request/schema/comparisonDocument Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiComparePost/responses/200/schema/differences/items/detailsummary Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiComparePost/responses/200/schema/differences/items/detail/items/Document Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiComparePost/responses/200/schema/differences/items/detail/items/Summary Source: NewSwagger
##[error]The path template has been changed for the operation. ValuePath: operation/V1AiCompareResult Source: NewSwagger
##[error]The required parameter has been added. ValuePath: operation/V1AiCompareResult/parameter/jobId Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiCompareResult/responses/default/schema/type Source: NewSwagger

**Comment by danmcpherson** [comment]
Confirming that existing flows will not break, I’m aware and managed these changes.

I’ve updated the spelling/typos.
Daniel

From: vmanoharas ***@***.***>
Date: Friday, 20 September 2024 at 11:22
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: Daniel McPherson ***@***.***>, Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] Sync changes from release-package fork V1.1.1 (PR #3656)

@vmanoharas commented on this pull request.

Hello @danmcpherson<https://github.com/danmcpherson>,

Kindly address review comments. Additionally, please see the swagger validation errors below and confirm that the existing flows would break or not?

##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/V1ConversionConvertPost/parameter/request/schema/configuration/documentResponseFormat Source: NewSwagger
##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/V1AiSummaryPost/parameter/request/schema/configuration/targetWords Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryPost/responses/200/schema/Type Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryPost/responses/200/schema/Content Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryPost/responses/200/schema/Length Source: NewSwagger
##[error]The path template has been changed for the operation. ValuePath: operation/V1AiSummaryResult Source: NewSwagger
##[error]The required parameter has been added. ValuePath: operation/V1AiSummaryResult/parameter/jobId Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryResult/responses/200/schema/Type Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryResult/responses/200/schema/Content Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryResult/responses/200/schema/TargetWords Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiSummaryResult/responses/200/schema/Length Source: NewSwagger
##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/V1AiComparePost/parameter/request/schema/sourceDocument Source: NewSwagger
##[error]The required input field has been added. Existing flows will not send this new field. ValuePath: operation/V1AiComparePost/parameter/request/schema/comparisonDocument Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiComparePost/responses/200/schema/differences/items/detailsummary Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiComparePost/responses/200/schema/differences/items/detail/items/Document Source: NewSwagger
##[error]The output field has been deleted. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiComparePost/responses/200/schema/differences/items/detail/items/Summary Source: NewSwagger
##[error]The path template has been changed for the operation. ValuePath: operation/V1AiCompareResult Source: NewSwagger
##[error]The required parameter has been added. ValuePath: operation/V1AiCompareResult/parameter/jobId Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/V1AiCompareResult/responses/default/schema/type Source: NewSwagger

________________________________

In certified-connectors/Tendocs Documents/apiDefinition.swagger.json<https://github.com/microsoft/PowerPlatformConnectors/pull/3656#discussion_r1768243288>:

> +            "schema": {

+              "$ref": "#/definitions/TemplateResponse"

+            }

+          },

+          "403": {

+            "description": "The request is not authorized to make the template request"

+          },

+          "404": {

+            "description": "Service has not been found, please contact ***@***.***"

+          },

+          "500": {

+            "description": "Something has gone wrong, please contact ***@***.***"

+          }

+        },

+        "summary": "Create a new document from a template and JSON",

+        "description": "Create a new Office document by merging an office template with JSON Obeject."

Please correct the typo "Obeject"

________________________________

In certified-connectors/Tendocs Documents/apiDefinition.swagger.json<https://github.com/microsoft/PowerPlatformConnectors/pull/3656#discussion_r1768245959>:

> +          "type": "string",

+          "example": "Summary",

+          "x-ms-summary": "Type",

+          "description": "The type of summary provided."

+        },

+        "content": {

+          "type": "string",

+          "example": "Summary of the document provided.",

+          "x-ms-summary": "Summary",

+          "description": "Summary of the document."

+        },

+        "targetWords": {

+          "type": "number",

+          "example": 300,

+          "x-ms-summary": "Target",

+          "description": "The targetted length in words."

Please correct the typo "targetted"

________________________________

In certified-connectors/Tendocs Documents/apiDefinition.swagger.json<https://github.com/microsoft/PowerPlatformConnectors/pull/3656#discussion_r1768252426>:

> +          "example": {

+            "introduction": "This document describes a set of new health and safety protocols which makes it critical all staff have reviewed."

+          }

+        },

+        "logoUrl": {

+          "type": "string",

+          "description": "Full URL to a logo that is displayed in the top left of the document.",

+          "x-ms-visibility": "advanced",

+          "x-ms-summary": "Document Logo",

+          "example": {

+            "logoUrl": "https://app.tendocs.com/images/doc.png"

+          }

+        },

+        "email": {

+          "type": "string",

+          "description": "Email address of the document recipeint.",

Please correct the typo

________________________________

In certified-connectors/Tendocs Documents/apiDefinition.swagger.json<https://github.com/microsoft/PowerPlatformConnectors/pull/3656#discussion_r1768255602>:

> +          "example": {

+            "checkboxText": "By checking this box you agree that you have read the updated Health and Safety protocols."

+          }

+        },

+        "expiry": {

+          "type": "string",

+          "description": "The date after which the document is no longer valid and cannot be accessed.",

+          "x-ms-visibility": "important",

+          "x-ms-summary": "Expiry Date",

+          "example": {

+            "expiry": "2024-06-01T09:00:00Z"

+          }

+        },

+        "isSignatureRequired": {

+          "type": "boolean",

+          "description": "Determines if the recipents name and an empty text box where they need to enter their name is included on the document page.",

Please correct the recipents

________________________________

In certified-connectors/Tendocs Documents/apiDefinition.swagger.json<https://github.com/microsoft/PowerPlatformConnectors/pull/3656#discussion_r1768258558>:

> +          "example": {

+            "expiry": "2024-06-01T09:00:00Z"

+          }

+        },

+        "isSignatureRequired": {

+          "type": "boolean",

+          "description": "Determines if the recipents name and an empty text box where they need to enter their name is included on the document page.",

+          "x-ms-visibility": "important",

+          "x-ms-summary": "Signature Required",

+          "example": {

+            "isSignatureRequired": true

+          }

+        },

+        "organisationTitle": {

+          "type": "string",

+          "description": "The title of the company, department or organisation that is repsonsible for sending the document.",

Please correct the type "repsonsible"

________________________________

In certified-connectors/Tendocs Documents/apiDefinition.swagger.json<https://github.com/microsoft/PowerPlatformConnectors/pull/3656#discussion_r1768260789>:

> +          }

+        },

+        "isSignatureRequired": {

+          "type": "boolean",

+          "description": "Determines if the recipents name and an empty text box where they need to enter their name is included on the document page.",

+          "x-ms-visibility": "important",

+          "x-ms-summary": "Signature Required",

+          "example": {

+            "isSignatureRequired": true

+          }

+        },

+        "organisationTitle": {

+          "type": "string",

+          "description": "The title of the company, department or organisation that is repsonsible for sending the document.",

+          "x-ms-visibility": "important",

+          "x-ms-summary": "Organsiation Title",

Please correct the typo "Organsiation"

________________________________

In certified-connectors/Tendocs Documents/apiDefinition.swagger.json<https://github.com/microsoft/PowerPlatformConnectors/pull/3656#discussion_r1768262191>:

> +          "example": {

+            "isSignatureRequired": true

+          }

+        },

+        "organisationTitle": {

+          "type": "string",

+          "description": "The title of the company, department or organisation that is repsonsible for sending the document.",

+          "x-ms-visibility": "important",

+          "x-ms-summary": "Organsiation Title",

+          "example": {

+            "organisation": "Human Resources"

+          }

+        },

+        "organisationWebsite": {

+          "type": "string",

+          "description": "Link to a webiste. This could be internal, or external to the organisation and provide more details on the document or process.",

Please correct the type "webiste"

________________________________

In certified-connectors/Tendocs Documents/apiDefinition.swagger.json<https://github.com/microsoft/PowerPlatformConnectors/pull/3656#discussion_r1768263916>:

> +          }

+        },

+        "organisationContactName": {

+          "type": "string",

+          "description": "Name of the owner who is responsible for the document or process.",

+          "x-ms-visibility": "important",

+          "x-ms-summary": "Organisation Owner",

+          "example": {

+            "organisationContactName": "Pauline Jones"

+          }

+        },

+        "organisationContactTitle": {

+          "type": "string",

+          "description": "Job title of the owner who is responsible for the document or process.",

+          "x-ms-visibility": "advanced",

+          "x-ms-summary": "Oragnisation Owner Title",

Please correct the typo "Oragnisation"

**Comment by danmcpherson** [comment]
Fixed and committed.

**Review by vmanoharas — APPROVED** [review]

**Review by vmanoharas — APPROVED** [review]
Dear Partner, 

Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. Read the blog here- [Announcing Partner Center to certify and publish Power Platform Copilot Connectors and Plugins - Microsoft Power Platform Blog](https://www.microsoft.com/en-us/power-platform/blog/power-automate/announcing-partner-center-to-certify-and-publish-power-platform-copilot-connectors-and-plugins/)

As next steps: 

1.	Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
2.	Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
3.	Validate the package for structure before submitting the package for certification to Partner Center - [Run Package Validator tool](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission%23step-7-validate-the-package-for-structure&data=05%7C02%7Cshefaali.patankar%40microsoft.com%7C6f0c4adf688f42a7107908dcb6c72a8c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638586213737574171%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=9QGRIRe%2FZ%2B1sgUF5JQjY2DeuyxDt8f8lDwpGFEaaBmg%3D&reserved=0) 
4.	Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
5.	Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0) and also [1000 Marketplace policies](https://learn.microsoft.com/en-us/legal/marketplace/certification-policies#100-general)(as applicable) 
6.	Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
7.	If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)

For any queries or concerns please get in touch with us on [certificationteam@microsoft.com](mailto:certificationteam@microsoft.com)

Many thanks,
Power Platform 
Copilot connector & plugin certification team

**Pull request merged**